### ISSUE TITLE
Dashboard stopped

### ISSUE THREAD

**Issue by skanda890** [microsoft/devhome#1736]
### Dev Home version

0.601.285.0

### Windows build number

10.0.22621.2428

### Other software

OS Build Version: 10.0.22621.2428.amd64fre.ni_release.220506-1250
.NET Version: .NET 6.0.22

### Steps to reproduce the bug

Go to Dashboard>Add New Widget>Select on Issues>Click on 'Pin'. Note: For this to happen, your account should not be linked.

### Expected result

It should pin the widget.

### Actual result

It got stuck.

### Included System Information

Physical Memory: 11.89GB (3.79GB free)
Processor Architecture: x64

### Included Extensions Information

Extensions: 
Microsoft.Windows.DevHome_0.601.285.0_x64__8wekyb3d8bbwe
Microsoft.Windows.DevHomeGitHubExtension_0.600.283.0_x64__8wekyb3d8bbwe

**Comment by skanda890** [comment]
This issue happened after #1729 happened.

**Comment by krschau** [comment]
Duplicate of #1730
